library/oneview_scope.py

```diff
@@ -145,6 +145,9 @@ def execute_module(self):
             return self.__update_resource_assignments()
 
     def __update_resource_assignments(self):
+        scope_res = self.get_by_name(self.data.get('name'))
```

Comment:
Instead of this, you can simply use 
if self.current_resource is None:
raise OneViewModuleResourceNotFound(self.MSG_RESOURCE_NOT_FOUND)